src/extension.ts

```diff
@@ -499,6 +500,72 @@ function launchMetals(
       });
     });
 
+    registerCommand("metals.new-scala-file", (dirArg: Uri) => {
+      const directory = (() => {
+        if (dirArg !== undefined) {
+          return dirArg.toString();
```

Comment:
Is there a more specific method than `toString()`? Like `fsPath` or similar

Reply:
fsPath is not URI (it misses `file:///` part).

Reply:
Ok, reading the docs `toString()` is actually the right method 👍